examples/rasterize_svg_to_png.js

```diff
@@ -15,25 +15,23 @@ const transloadit = new Transloadit({
 
 const filePath = process.argv[2]
 
-;(async () => {
-  try {
-    const status = await transloadit.createAssembly({
-      files: {
-        file1: filePath,
-      },
-      params: {
-        steps: {
-          png: {
-            use: ':original',
-            robot: '/image/resize',
-            format: 'png',
-          },
+try {
+  const status = await transloadit.createAssembly({
+    files: {
+      file1: filePath,
+    },
+    params: {
+      steps: {
+        png: {
+          use: ':original',
+          robot: '/image/resize',
+          format: 'png',
         },
       },
-      waitForCompletion: true,
-    })
-    console.log('Your PNG file:', status.results.png[0].url)
-  } catch (err) {
-    console.error('createAssembly failed', err)
-  }
-})()
+    },
+    waitForCompletion: true,
+  })
+  console.log('Your PNG file:', status.results.png[0].url)
+} catch (err) {
+  console.error('createAssembly failed', err)
+}
```

Comment:
I consider this the same as my other comment about try/catch blocks. This does log an additional string, but I don’t think we need it.